.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: node_js
 node_js:
-  - 7.10.0
+  - 8
+  - node
```

Comment:
Travis says `node` implies the latest stable node release. Is this redundant given `8` above?